Workbooks/Virtual Machine Scale Sets - Performance Analysis/Performance Analysis (Perf)/settings.json

```diff
@@ -1,9 +1,9 @@
 {
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
-    "name":"Performance",
+    "name":"Perf Counters",
     "author": "Microsoft",
     "galleries": [
-        { "type": "workbook", "resourceType": "microsoft.compute/virtualmachinescalesets", "categoryKey": "vmInsightsVirtualMachinesPerf", "order": 200 },
-        { "type": "insights", "resourceType": "microsoft.compute/virtualmachinescalesets", "categoryKey": "vmInsightsVirtualMachinesPerf", "order": 200 }
+        { "type": "workbook", "resourceType": "microsoft.compute/virtualmachinescalesets", "categoryKey": "vmInsightsVirtualMachines", "order": 210 },
```

Comment:
do we need to define the category key "vmInsightsVirtualMachines"?

Reply:
Thanks @yves-chan, resolved.